app/views/shared/_lessons_table.slim

```diff
@@ -18,3 +20,7 @@ table class='mdl-data-table mdl-js-data-table resource-table'
         = link_to lesson.date, lesson_path(lesson)
       td class='mdl-data-table__cell--non-numeric'
         = link_to lesson.subject.subject_name, lesson_path(lesson)
+      td class='mdl-data-table__cell--numeric'
+        = link_to lesson.group.students.count, group_path(lesson.group)
+      td class='mdl-data-table__cell--numeric'
```

Comment:
No need for the class, it's considered numeric by default unless specified as non-numeric